### ISSUE TITLE
Unused sniff false positive for class constants & static variables

### ISSUE THREAD

**Issue by iquito** [slevomat/coding-standard#1225]
In the newest version (7.0.8) classes are reported as unused when they are only used for their constants or static variables, something like:

```php
use Symfony\Component\Routing\Generator\UrlGeneratorInterface;
use Symfony\Component\Routing\RouterInterface;

class SomeController extends AbstractController
{
    public function generateRoute(RouterInterface $router): string
    {
        return $router->generate('login_password_set', [
            'resetCode' => 'somecode',
        ], referenceType: UrlGeneratorInterface::RELATIVE_PATH);
    }
}
```

This reports:

```
Type Symfony\Component\Routing\Generator\UrlGeneratorInterface is not used in this file.
```

The same happens when using static variables from classes, like `SomeClass::$variable` - SomeClass will be marked as unused. Some change in version `7.0.2` seems to have introduced this, as when I revert to `7.0.1` these issues disappear, and anything over `7.0.2` shows this behavior up to the current version.

**Comment by kukulich** [comment]
I'm sorry I'm not able to reproduce it: https://github.com/slevomat/coding-standard/commit/29bbdc94fda5e5d2cd2c7a6ade551736a8ac39f7

**Comment by iquito** [comment]
So weird, I cannot reproduce it anymore either, but two days ago it was very reproducible (and removed a few use statements when running `phpcbf`, leading to errors in development). Not sure how that is possible, but I guess it is not a problem anymore. Thanks!
